### PR TITLE
dont add new tracks after 18pm

### PR DESCRIPTION
**Pull request by radeklos**

### CODE REVIEW

**Comment by radeklos** [comment]
@krak3n @peeklondon CR please.

File: fm_api/api.go [hunk 1]
```diff
@@ -65,14 +66,16 @@ func (m *FmApiManager) GetQueue() ([]QueueItem, error) {
 func (m *FmApiManager) Listen(c chan bool, l int, r func() string) {
 	for {
 		<-c
-		queue, err := m.GetQueue()
-		if err != nil {
-			log.Printf("Error when fetching queue", err)
-			continue
-		}
+		if time.Now().Hour() <= 18 { // dont run after 6pm
```
Comment (gregatsoon) [inline fm_api/api.go:69]: Do you need anything for after 9am? Also, what happens at weekends?

Comment (krak3n) [inline fm_api/api.go:69]: Yup.

So this should only run between the hours of 9am to 6pm and Mon - Fri.

**Comment by radeklos** [comment]
@peeklondon It plays only when a track finishes.. So it shouldn't play in weekends and bank holidays if someone don't add track into the queue.

File: fm_api/api.go (hunk 1 above)
Comment (radeklos) [inline fm_api/api.go:69]: It plays only when a track finishes.. So it shouldn't play in weekends and bank holidays if someone don't add track into the queue.

Comment (krak3n) [inline fm_api/api.go:69]: Ah I see, ok cool 👍

**Pull request merged**